src/System.Collections.Concurrent/tests/BlockingCollectionTests.cs

```diff
@@ -282,9 +278,7 @@ private static void Test2_ConcurrentAdd(int numOfThreads, int numOfElementsPerTh
 
                     for (int j = startOfSequence; j < endOfSequence; ++j)
                     {
-                        if (!blockingCollection.TryAdd(j))
-                            Assert.False(true,
-                                String.Format("Test2_ConcurrentAdd(numOfThreads={0}, numOfElementsPerThread={1}): > test failed - TryAdd returned false unexpectedly", numOfThreads, numOfElementsPerThread));
+                        Assert.True(blockingCollection.TryAdd(j));
```

Comment:
Updating these tests to use InlineData instead of passing via method call.  The entire set probably needs to be gone over for this, though.  Future intern work.